### ISSUE TITLE
Document backwards incompatible changes, like --chain -> --keep-chain

### ISSUE THREAD

**Issue by speleo3** [Electrostatics/pdb2pqr#173]
I just noticed that the PROPKA 2.1 `--chain` argument is now `--keep-chain` (changed in 6f733072be5ee6dfdbe841eb0a8ab7bb7c9b9d24), and `--chain` now takes an argument and has a different meaning (is handled by `PROPKA`). This is neither mentioned in https://github.com/Electrostatics/pdb2pqr/blob/master/ChangeLog.md nor in https://github.com/Electrostatics/pdb2pqr/blob/master/docs/releases.rst

**Comment by sobolevnrm** [comment]
I agree.  I will make the change when I have the time... unless anyone else beats me to it. :)

Thanks!
